enterprise/ha/src/test/java/org/neo4j/test/ha/ClusterTest.java

```diff
@@ -59,8 +60,8 @@ public void testCluster() throws Throwable
     {
         ClusterManager clusterManager = new ClusterManager( fromXml( getClass().getResource( "/threeinstances.xml" ).toURI() ),
                 testDirectory.directory(  "testCluster" ),
-                MapUtil.stringMap( HaSettings.ha_server.name(), "localhost:6001-6005",
-                                  HaSettings.tx_push_factor.name(), "2"));
+                entry( HaSettings.ha_server.name(), "localhost:6001-6005" ).
+                        entry( HaSettings.tx_push_factor.name(), "2" ).create() );
```

Comment:
Looks like a nice idiom worth spreading, but perhaps not in this PR. I don't know. It irks me a little that there's only one usage of it right now.